### PR TITLE
Add new developer (Akeel) 

### PR DESCRIPTION
**Pull request by akeel230**
## Welcome PR details 

- SEF Hive User Name
Akeel_malik

- Introduce yourself thread entry url
https://sef.discourse.group/t/welcome-please-introduce-yourself/7/63

- How many Pull Requests have you sent to sef-repos? (Check only one)
  - [ ] This is the first one 
  - [ ] 2 Including this
  - [ ] 3 - 4 Including this
  - [x] 5 or more Including this 

## Approach
added the info in to the developer.Json file via the clone from the sef-dev repo 

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by Piumal1999 — APPROVED** [review]

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]
Hey, @akeel230 would you be able to resolve the conflicts to get this merged?

**Comment by akeel230** [comment]
@sure @Gravewalker666 ill resolve the conflict and push it to the PR

**Comment by akeel230** [comment]
@Gravewalker666 conflict resolved take a look

**Comment by akeel230** [comment]
Screenshot (179)
here u go please take a look @Gravewalker666

File: developers.json [hunk 1]
```diff
@@ -34,6 +34,12 @@
         "welcomeThreadEntryUrl": "https://sef.discourse.group/t/welcome-please-introduce-yourself/7/20?u=gravewalker",
         "quote": "In my free time, I travel with my friends, do camping in various places in Sri Lanka."
     },
+    {
+
```
Comment (anjula-sack) [inline developers.json:38]: Remove this empty line

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: developers.json (hunk 1 above)
Comment (akeel230) [reply developers.json]: @anjula-sack done. Thanks

**Review by anjula-sack — APPROVED** [review]

**Pull request closed without merging**